[master] test run fails

I'm in the process of creating a spec file to produce an rpm that conforms to common filesystem standards. I'm almost done and the everything works fine except that I'd like to hook up the tests as well but they fail. The first thing I see is this:

```
[dennis@nexus influxdb]$ go test -v ./...
# github.com/influxdb/influxdb/influxql
influxql/functions_test.go:8:2: cannot find package "github.com/davecgh/go-spew/spew" in any of:
	/usr/lib/golang/src/github.com/davecgh/go-spew/spew (from $GOROOT)
	/home/dennis/gocode/src/github.com/davecgh/go-spew/spew (from $GOPATH)
FAIL	github.com/influxdb/influxdb/influxql [setup failed]
# github.com/influxdb/influxdb/meta
influxql/functions_test.go:8:2: cannot find package "github.com/davecgh/go-spew/spew" in any of:
	/usr/lib/golang/src/github.com/davecgh/go-spew/spew (from $GOROOT)
	/home/dennis/gocode/src/github.com/davecgh/go-spew/spew (from $GOPATH)
FAIL	github.com/influxdb/influxdb/meta [setup failed]
# github.com/influxdb/influxdb/services/graphite
influxql/functions_test.go:8:2: cannot find package "github.com/davecgh/go-spew/spew" in any of:
	/usr/lib/golang/src/github.com/davecgh/go-spew/spew (from $GOROOT)
	/home/dennis/gocode/src/github.com/davecgh/go-spew/spew (from $GOPATH)
FAIL	github.com/influxdb/influxdb/services/graphite [setup failed]
# github.com/influxdb/influxdb/services/opentsdb
influxql/functions_test.go:8:2: cannot find package "github.com/davecgh/go-spew/spew" in any of:
	/usr/lib/golang/src/github.com/davecgh/go-spew/spew (from $GOROOT)
	/home/dennis/gocode/src/github.com/davecgh/go-spew/spew (from $GOPATH)
FAIL	github.com/influxdb/influxdb/services/opentsdb [setup failed]
```

I can fix this manually by running "go get github.com/davecgh/go-spew/spew" but that is not mentioned in the build instructions.

A bit later I see this:
```
=== RUN TestPointsWriter_WritePoints
[write] 2015/06/11 22:41:32 write failed for shard 13 on node 1: a failure
[write] 2015/06/11 22:41:32 write failed for shard 13 on node 2: a failure
[write] 2015/06/11 22:41:32 write failed for shard 13 on node 3: a failure
[write] 2015/06/11 22:41:32 write failed for shard 11 on node 1: a failure
[write] 2015/06/11 22:41:32 write failed for shard 11 on node 2: a failure
[write] 2015/06/11 22:41:32 write failed for shard 11 on node 3: a failure
[write] 2015/06/11 22:41:32 write failed for shard 15 on node 1: a failure
[write] 2015/06/11 22:41:32 write failed for shard 17 on node 1: a failure
[write] 2015/06/11 22:41:32 write failed for shard 23 on node 2: a failure
[write] 2015/06/11 22:41:32 write failed for shard 25 on node 2: a failure
[write] 2015/06/11 22:41:32 write failed for shard 27 on node 2: a failure
[write] 2015/06/11 22:41:32 write failed for shard 27 on node 3: a failure
[write] 2015/06/11 22:41:32 write failed for shard 29 on node 2: a failure
[write] 2015/06/11 22:41:32 write failed for shard 29 on node 3: a failure
[write] 2015/06/11 22:41:32 write failed for shard 31 on node 1: a failure
[write] 2015/06/11 22:41:32 write failed for shard 31 on node 2: a failure
[write] 2015/06/11 22:41:32 write failed for shard 33 on node 1: a failure
[write] 2015/06/11 22:41:32 write failed for shard 33 on node 2: a failure
[write] 2015/06/11 22:41:32 write failed for shard 35 on node 3: a failure
[write] 2015/06/11 22:41:32 write failed for shard 37 on node 3: a failure
[write] 2015/06/11 22:41:32 write failed for shard 43 on node 1: a failure
[write] 2015/06/11 22:41:32 write failed for shard 43 on node 2: a failure
[write] 2015/06/11 22:41:32 write failed for shard 43 on node 3: a failure
[write] 2015/06/11 22:41:32 write failed for shard 45 on node 1: a failure
[write] 2015/06/11 22:41:32 write failed for shard 45 on node 2: a failure
[write] 2015/06/11 22:41:32 write failed for shard 45 on node 3: a failure
[write] 2015/06/11 22:41:32 write failed for shard 47 on node 1: a failure
[write] 2015/06/11 22:41:32 write failed for shard 49 on node 1: a failure
--- PASS: TestPointsWriter_WritePoints (0.00s)
```
It looks like there are errors yet the test seems to pass anyway. Is this intentional or is there a buf in this test?

--- a/ui/src/pageLayout/components/CloudNav.tsx
+++ b/ui/src/pageLayout/components/CloudNav.tsx
@@ -1,0 +1,43 @@
+import React, {PureComponent} from 'react'
+
+// Types
+import {IconFont} from 'src/clockface'
+
+export default class CloudNav extends PureComponent {
+  render() {
+    if (!this.shouldRender) {
+      return null
+    }
+
+    return (
+      <div className="nav--item">
+        <a className="nav--item-icon">
+          <span className={`icon sidebar--icon ${IconFont.CuboNav}`} />
+        </a>
+        <div className="nav--item-menu">
+          <a className="nav--item-header" href="">
+            Cloud
+          </a>
+          <a className="nav--sub-item" href={this.usageURL}>
+            Usage
+          </a>
+          <a className="nav--sub-item" href={this.billingURL}>
+            Billing
+          </a>
+        </div>
+      </div>
+    )
+  }
+
+  private get shouldRender(): boolean {
+    return process.env.CLOUD === 'true'
+  }
+
+  private get usageURL(): string {
+    return `${process.env.CLOUD_URL}${process.env.CLOUD_USAGE_PATH}`
+  }
+
+  private get billingURL(): string {
+    return `${process.env.CLOUD_URL}${process.env.CLOUD_BILLING_PATH}`
+  }
+}

--- a/ui/src/pageLayout/components/NavMenu.tsx
+++ b/ui/src/pageLayout/components/NavMenu.tsx
@@ -5,7 +5,6 @@ import _ from 'lodash'
 // Components
 import NavMenuItem from 'src/pageLayout/components/NavMenuItem'
 import NavMenuSubItem from 'src/pageLayout/components/NavMenuSubItem'
-import {Select} from 'src/clockface'
 
 // Types
 import {ErrorHandling} from 'src/shared/decorators/errors'
@@ -28,7 +27,7 @@ class NavMenu extends PureComponent<Props> {
 
     return (
       <nav className="nav" data-testid="nav">
-        <Select type={NavMenuItem}>{children}</Select>
+        {children}
       </nav>
     )
   }

--- a/ui/src/pageLayout/containers/Nav.tsx
+++ b/ui/src/pageLayout/containers/Nav.tsx
@@ -5,6 +5,7 @@ import {connect} from 'react-redux'
 
 // Components
 import NavMenu from 'src/pageLayout/components/NavMenu'
+import CloudNav from 'src/pageLayout/components/CloudNav'
 
 // Types
 import {MeState, AppState} from 'src/types/v2'
@@ -12,17 +13,15 @@ import {IconFont} from 'src/clockface'
 
 import {ErrorHandling} from 'src/shared/decorators/errors'
 
-interface Props extends WithRouterProps {
+interface OwnProps {
   isHidden: boolean
   me: MeState
 }
 
+type Props = OwnProps & WithRouterProps
+
 @ErrorHandling
 class SideNav extends PureComponent<Props> {
-  constructor(props) {
-    super(props)
-  }
-
   public render() {
     const {isHidden, me} = this.props
     const {location} = this.props
@@ -118,6 +117,7 @@ class SideNav extends PureComponent<Props> {
             highlightPaths={['tokens_tab']}
           />
         </NavMenu.Item>
+        <CloudNav />
       </NavMenu>
     )
   }
@@ -130,4 +130,4 @@ const mstp = (state: AppState) => {
   return {isHidden, me}
 }
 
-export default connect(mstp)(withRouter(SideNav))
+export default connect(mstp)(withRouter<OwnProps>(SideNav))

